web/client/components/misc/datetimepicker/__tests__/DateTimePicker-test.js

```diff
@@ -60,7 +60,23 @@ describe('DateTimePicker component', () => {
         const calendar = container.querySelector('.rw-calendar-popup');
         expect(calendar.style.display).toBe('block');
     });
-
+    it('calendar opens to today when value is null or undefined', function() {
+        ReactDOM.render(<DateTimePicker/>, document.getElementById("container"));
+        const container = document.getElementById('container');
+        const button = container.querySelector('.rw-btn-calendar');
+        TestUtils.Simulate.click(button);
+        const monthLabel = document.querySelector('#container > div > div.rw-calendar-popup.rw-popup-container .rw-btn-view');
+        expect(monthLabel.innerHTML).toBe(moment().format('MMMM YYYY'));
```

Comment:
no check on today date

Reply:
`moment()` returns the today date. 
We can check only that the calendar opens at the month of "today", because it is a montly calendar.